edx_event_bus_kafka/internal/consumer.py

```diff
@@ -180,11 +174,9 @@ class ConsumeEventsCommand(BaseCommand):
     example:
         python3 manage.py cms consume_events -t user-event-debug -g user-event-consumers
             -s org.openedx.learning.auth.session.login.completed.v1
-
-    # TODO (EventBus): Add pointer to relevant future docs around topics and consumer groups, and potentially
-    update example topic and group names to follow any future naming conventions.
-
     """
+    # TODO: Add pointer to relevant future docs around topics and consumer groups, and potentially
+    # update example topic and group names to follow any future naming conventions.
```

Comment:
Should we just replace this with a link to https://openedx.atlassian.net/wiki/spaces/AC/pages/3508699151/How+to+start+using+the+Event+Bus for a starting place for documentation?

Reply:
I'm not sure it's worth linking any particular place until we have something written about naming conventions.

Reply:
Then maybe best to drop the todo as well. We could add a todo to the doc for address naming conventions if we think this is important, or we can just leave it until someone asks.